commands/list_clusters.go

```diff
@@ -124,6 +124,7 @@ func clustersTable(args listClustersArguments) (string, error) {
 		color.CyanString("ID"),
 		color.CyanString("ORGANIZATION"),
 		color.CyanString("NAME"),
+		color.CyanString("RELEASE"),
```

Comment:
Should we call this `RELEASE VERSION` as well @marian?

Reply:
II am fine with the brief version here.

Reply:
Then it would be nice to get an approval on this.